### PR TITLE
[enterprise-metrics] upgrade to gem 1.5.0

### PR DESCRIPTION
**Pull request by trevorwhitney**
Signed-off-by: Trevor Whitney <trevorjwhitney@gmail.com>

### CODE REVIEW

**Review by jdbaldry — DISMISSED** [review]
LGTM, assuming there are no breaking configuration changes that need to be incorporated.

May also be worth updating the exports with `$ make exports`

**Comment by trevorwhitney** [comment]
breaking changes in the changelog list oidc and the read/write permissions, so nothing that should affect this helm chart. I will run the make exports command before merging, thanks!

**Review by jdbaldry — APPROVED** [review]
LGTM

**Pull request merged**
